.github/workflows/deploy_preview.yml

```diff
@@ -9,7 +9,7 @@ permissions:
 
 jobs:
   deploy-preview:
-    if: ${{ github.repository == 'primer/react' }}
+    if: ${{ github.event.pull_request.head.repo.full_name == 'primer/react' }} # source repository for pull_request
```

Comment:
if it's a fork, skip the default deploy

Reply:
Could you also add this as a code comment to the yml file?

Reply:
done :)